osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -77,13 +77,13 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
                     wideAngleBonus = calcWideAngleBonus(currAngle);
                     acuteAngleBonus = calcAcuteAngleBonus(currAngle);
 
-                    if (osuCurrObj.StrainTime > 100) // Only buff deltaTime exceeding 300 bpm 1/2.
+                    if (osuCurrObj.StrainTime > 100 || osuCurrObj.LazyJumpDistance < 50) // Only buff deltaTime exceeding 300 bpm 1/2 and spacing exceeding one radius (to ensure easily cheesable patterns aren't buffed)
```

Comment:
could this be using `OsuDifficultyHitObject.NORMALISED_RADIUS` instead of hardcoding `50` here?

Reply:
this check was actually redundant, so I just removed it